integrations/discord-bot/voxguard/dashboard/static/style.css

```
@@ -0,0 +1,403 @@
+.server-card:hover {
+  background: var(--glass-hi); border-color: var(--glass-edge); transform: translateY(-2px);
+}
+.server-top { display: flex; align-items: center; gap: 12px; margin-bottom: 13px; }
+.server-icon {
+  width: 42px; height: 42px; border-radius: 12px; flex: none;
+  background: rgba(255,255,255,.08); border: 1px solid var(--glass-line);
+  display: grid; place-items: center;
+  font-weight: 600; font-size: 14px; color: var(--text); object-fit: cover;
+  letter-spacing: -.02em;
+}
+.server-name { font-weight: 600; font-size: 14px; line-height: 1.3; word-break: break-word; }
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)